File: reactor/r2dbc-mysql-webclient/src/main/java/com/web/client/demo/exception/ClientFaultInfo.java
package com.web.client.demo.exception;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@NoArgsConstructor
@Data
public class ClientFaultInfo {

//    private LocalDateTime timestamp;
    private String path;
    private String error;
    private String requestId;
    private String status;
    private String errorCd;
    private String errorMsg;

}

File: kotlin/kotlin-r2dbc-mysql/kotlin-r2dbc-mysql-webclient/src/main/java/ua/mai/zyme/r2dbcmysql/webclient/exception/ClientFaultInfo.kt
package ua.mai.zyme.r2dbcmysql.webclient.exception

import java.time.LocalDateTime

data class ClientFaultInfo(
//    var timestamp: LocalDateTime? = null,
    var path: String? = null,
    var error: String? = null,
    var requestId: String? = null,
    var status: String? = null,
    var errorCd: String? = null,
    var errorMsg: String? = null
)